### DISCUSSION TITLE
Tina Cloud no access error

### DISCUSSION THREAD

**Discussion by jflores353** [tinacms/tinacms#3941] [Help]
I've successfully built Tina CMS with my Jekyll site using Github Actions per the Tina docs. However, now when I try to navigate to my site path "/admin" I'm prompted to login but am shown "Error: No access. Access is currently blocked." I've logged into my Tina.io account and am clearly listed as a collaborator with access set to 'Editor'. What are my next steps to fix this?

**Comment by fabianmarz** [comment]
@jflores353, did you ever resolve this issue? I'm running in the same issue right now.

**Reply by kldavis4** [reply]
This can happen if you have logged in with both an email and a GitHub account and the wrong account is invited to the project.
